arguments.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import argparse
+
+
+def parameters():
+    parser = argparse.ArgumentParser(
+            prog='BetterWeather',
+            description="""This program is written in Python.
```

Comment:
You have already this info in the readme file, no need to say that here, especially since it's obvious to whoever will either execute it or look into the code;)

Reply:
Done.